xds/src/main/java/io/grpc/xds/CdsLoadBalancer2.java

```diff
@@ -206,8 +206,9 @@ private void handleClusterDiscovered() {
               }
               loopStatus = Status.UNAVAILABLE.withDescription(String.format(
                   "CDS error: circular aggregate clusters directly under %s for "
-                      + "root cluster %s, named %s",
-                  clusterState.name, root.name, namesCausingLoops));
+                      + "root cluster %s, named %s xDS node ID: %s",
```

Comment:
Add a comma between named and XDS node ID

Reply:
Done